New Notification Window per user

rcapecce opened the pull request:
Resolve #3927

e-Evolution reviewed (changes requested):
@rcapecce can please set the ReleaseNo="" into the XML Script and check if are with right sequence to accept this PR

rcapecce commented:
> @rcapecce can please set the ReleaseNo="" into the XML Script and check if are with right sequence to accept this PR

Hi @e-Evolution , sure, I corrected the xml sequences and added the releaseNo, you can already see the change.
